### DISCUSSION TITLE
Helm chart for deployment

### DISCUSSION THREAD

**Discussion by nenkoru** [Freika/dawarich#286] [Ideas]
Hey there!

I wonder if it would be a good idea to create and maintain a Helm chart for deploying datawarich. For instance there is a TrueCharts [1] which is used among self-hosters who use TrueNAS.
I am personally running a k8s as a selfhoster.

[1] https://truecharts.org

**Comment by Cogitri** [comment]
Hej, I have created a Helm-Chart for dawarich, feel free to take a look: https://github.com/Cogitri/charts/tree/master/charts/dawarich :)

**Reply by jdblack** [reply]
Hehehe, I came here to mention this =)  Thank you very much!

**Reply by Cogitri** [reply]
You're welcome, glad that it's of use to you :)
